src/showcase-merge/app/shared/component-viewer/one-variant-only-notification/one-variant-only-notification.component.html

```diff
@@ -0,0 +1,14 @@
+<sbb-notification
+  *ngIf="variantOnly"
+  type="info"
+  class="sbb-one-variant-only-notification"
+  readonly
+>
+  <span class="sbb-one-variant-only-notification-title">{{ variantOnly }}-only component</span>
```

Comment:
`component` doesn't really reflect what this is. Maybe reduce it to just {{ ... }}-only? Or what do you think?